<?xml version="1.0"?>
<launch>
  <!-- Parameters to pass to the ROS node -->
  <arg name="serial_port" default="/dev/for_docker/blacktool"/>
  
  <!-- This calls the ROS node we have written, using the parameters above -->
  <node name="nut_tool_server" pkg="o2as_precision_gripper" type="nut_tool_server.py" respawn="false" output="screen">
    <param name="serial_port" value="$(arg serial_port)"/>
    <rosparam file="$(find o2as_precision_gripper)/config/nut_tool_parameters.yaml" command="load" />
  </node>
</launch>
